dom.bs

```diff
@@ -2001,54 +2001,74 @@ can be used to explore this matter in more detail.
 
 <h3 id=node-trees>Node tree</h3>
 
-<p>{{Document}}, {{DocumentType}}, {{DocumentFragment}}, {{Element}}, {{Text}},
-{{ProcessingInstruction}}, and {{Comment}} objects (simply called
-<dfn export id=concept-node>nodes</dfn>) <a>participate</a> in a <a>tree</a>, simply named the
+<p><dfn export id=concept-node>Nodes</dfn> are objects that <a>implement</a> {{Node}}.
+<a for=/>Nodes</a> <a>participate</a> in a <a>tree</a>, which is known as the
 <dfn export id=concept-node-tree>node tree</dfn>.
 
-<p>A <a>node tree</a> is constrained as follows, expressed as a relationship between the type of
-<a>node</a> and its allowed <a>children</a>:
+<div class=note>
+ <p>In practice you deal with more specific objects.
+
+ <p>Objects that <a>implement</a> {{Node}} also implement an inherited interface: {{Document}},
+ {{DocumentType}}, {{DocumentFragment}}, {{Element}}, {{CharacterData}}, or {{Attr}}.
+
+ <p>Objects that implement {{DocumentFragment}} sometimes implement {{ShadowRoot}}.
+
+ <p>Objects that <a>implement</a> {{Element}} also typically implement an inherited interface, such
```

Comment:
The inconsistent linking of "implement" here is a bit confusing. I'd suggest either in every sentence in this note, or in just the first instance.